src/storage/mvcc/reader/reader.rs

```diff
@@ -118,6 +174,33 @@ impl<S: Snapshot> MvccReader<S> {
         Ok(res)
     }
 
+    pub fn load_write(&mut self, key: &Key, ts: TimeStamp) -> Result<Option<Write>> {
```

Comment:
I can't find any place using it.

Reply:
Sorry. This function was used by commit and I forgot to remove it.